src/test/cpp/gtester/ParsePopulationModel.cpp

```diff
@@ -0,0 +1,24 @@
+#include "pop/PopulationModel.h"
+#include <gtest/gtest.h>
+#include <boost/property_tree/ptree.hpp>
+#include <boost/property_tree/xml_parser.hpp>
+#include <boost/filesystem.hpp>
+#include <iostream>
+
+namespace Tests {
+
+TEST(ParsePopulationModel, ParseDefaultPopulationModel) {
```

Comment:
This test looks super cool. But what happens if we feed the parser a badly-formatted file or a file that doesn't define all `PopulationModel` fields? What kind of exception does that throw? Maybe you can write a test case for that, too.